Dockerfile

```diff
@@ -94,7 +201,8 @@ ARG TORCH_CUDA_ARCH_LIST="7.0 8.6"
 # Build vllm python package
 RUN --mount=type=cache,target=/root/.cache/pip \
     --mount=type=cache,target=/root/.cache/ccache \
-    MAX_JOBS=${MAX_JOBS} TORCH_CUDA_ARCH_LIST=${TORCH_CUDA_ARCH_LIST} \
+    MAX_JOBS=${MAX_JOBS} \
+    TORCH_CUDA_ARCH_LIST=${PYTORCH_ROCM_ARCH:-${TORCH_CUDA_ARCH_LIST}} \
```

Comment:
If PYTORCH_ROCM_ARCH is set, it will use that value
If PYTORCH_ROCM_ARCH is not set, it will fall back to the original TORCH_CUDA_ARCH_LIST